tests/testthat/test-metadata.R

```diff
@@ -1,3 +1,57 @@
+test_that("cache_data runs successfully", {
+  # Don't skip on CI
+  if (Sys.getenv("GITHUB_ACTIONS") == "true") {
+    cli::cli_alert_info("Running cache_data() in CI...")
+  } else {
+    skip_on_cran()
+  }
+
+  cache_folder <- get_cache_folder()
+  cli::cli_alert_info("Cache folder path: {cache_folder}")
+
+  expect_no_error(cache_data())
+
+  # Check that a file exists in cache (if not in CI)
+  if (Sys.getenv("GITHUB_ACTIONS") != "true") {
+    cached_files <- files_to_cache()
+    existing_files <- list.files(cache_folder, full.names = TRUE)
+    expect_true(any(basename(existing_files) %in% cached_files),
+                info = "At least one expected data file should exist in the cache"
+    )
+  }
```

Comment:
This could be written as a separate test with `skip_on_ci()` above it so it is only run when not doing Continuous Integration.

Reply:
Agreed - changed